static/app/views/replays/detail/detailLayout.tsx

```diff
@@ -120,47 +123,76 @@ function EventMetaData({
 
   return (
     <KeyMetrics>
-      <KeyMetricData
-        keyName={t('Timestamp')}
-        value={event ? <TimeSince date={event.dateReceived} /> : <HeaderPlaceholder />}
+      <ProjectBadge
+        project={{
+          slug: event?.projectSlug || '',
+          id: event?.projectID,
+          platform: event?.platform as PlatformKey,
+        }}
+        avatarSize={16}
```

Comment:
This should be a separate PR, but you probably need the `useProjects` hook to fetch the full project to use.